pkg/recipes/driver/bicep.go

```diff
@@ -403,3 +436,28 @@ func (d *bicepDriver) getGCOutputResources(current []string, previous []string)
 
 	return diff, nil
 }
+
+func (d *bicepDriver) FindSecretIDs(ctx context.Context, envConfig recipes.Configuration, definition recipes.EnvironmentDefinition) (secretStoreIDResourceKeys map[string][]string, err error) {
+	secretStoreIDResourceKeys = make(map[string][]string)
+	if envConfig.RecipeConfig.Bicep.Authentication != nil {
+		for _, v := range envConfig.RecipeConfig.Bicep.Authentication {
+			secretStoreIDResourceKeys[v.Secret] = []string{}
```

Comment:
Why is this set to `[]string{}`?

Reply:
here `[]string{}` represents the keys we look for in the secret store resource, and if it is empty it means `LoadSecrets()` will return all secrets in the secretstore resource

Reply:
we can also set it to nil